.github/workflows/ci.yml

```diff
@@ -100,6 +100,6 @@ jobs:
       - uses: volta-cli/action@v1
 
       - name: install dependencies
-        run: yarn install --no-lockfile
+        run: npm ci
```

Comment:
For the floating dependencies job, we specifically need to make sure it runs without a lockfile, try `npm ci --no-package-lock`.

Reply:
I thought I'd added that. WTF is going on...

Reply:
Fixed.